src/esthri/Cargo.toml

```diff
@@ -84,8 +84,9 @@ version = "0.3"
 optional = true
 
 [dependencies.tokio]
-version = "1.5"
-features = [ "io-std", "net", "rt", "time",]
+version = "1.6"
+features = [ "io-std", "net", "time", "rt",]
```

Comment:
Could `net` be moved to `http_server`? also I don't see `tokio::time` anywhere so maybe `time` can be removed

Reply:
Was able to remove net and time, I think we use `tokio::net` directly (probably do via warp though)